src/popup/bookmark.js

```diff
@@ -136,15 +135,16 @@ App.registerPanel(P_SAVING, {
   panelSelector: ".saving-panel",
 
   async url() {
-    const {endpoint, email, apitoken} = await App.getSettings("endpoint", "email", "apitoken")
+    const {endpoint} = await App.getSettings("endpoint")
 
-    return `${endpoint}/api/v1/bookmarks?auth_token=${email}:${apitoken}`
+    return `${endpoint}/api/v1/bookmarks`
```

Comment:
I want'd to say `unnecessary return statement` but

Reply:
actually, now that I think about it...when did javascript start getting written without semicolons? Reminds me of ye olde `bfd`

Reply:
:D